index.js

```diff
@@ -24,65 +24,81 @@ module.exports.subscribe = (event, callback) => {
 };
 
 // eventToBuild transforms pubsub event message to a build object.
-module.exports.eventToBuild = (data) => {
-  return JSON.parse(new Buffer(data, 'base64').toString());
-}
+module.exports.eventToBuild = (data) => JSON.parse(new Buffer(data, 'base64').toString());
 
 const DEFAULT_COLOR = '#4285F4'; // blue
 const STATUS_COLOR = {
-  'QUEUED': DEFAULT_COLOR,
-  'WORKING': DEFAULT_COLOR,
-  'SUCCESS': '#34A853', // green
-  'FAILURE': '#EA4335', // red
-  'TIMEOUT': '#FBBC05', // yellow
-  'INTERNAL_ERROR': '#EA4335', // red
+  QUEUED: DEFAULT_COLOR,
+  WORKING: DEFAULT_COLOR,
+  SUCCESS: '#34A853', // green
+  FAILURE: '#EA4335', // red
+  TIMEOUT: '#FBBC05', // yellow
+  INTERNAL_ERROR: '#EA4335', // red
 };
 
 // createSlackMessage create a message from a build object.
 module.exports.createSlackMessage = (build) => {
-  let message = {
-    text: `Build \`${build.id}\` finished`,
+  const buildFinishTime = new Date(build.finishTime);
+  const buildStartTime = new Date(build.startTime);
+
+  const timestamp = Math.round(buildFinishTime.getTime() / 1000);
```

Comment:
you probably want a different timestamp if `!isWorking`?

Reply:
Oh true! This should do it!